app/src/utils/readable-mime-type/types.json

```diff
@@ -222,7 +222,7 @@
 	"application/x-hdf": "Hierarchical Data Format",
 	"audio/vnd.rip": "Hit'n'Mix",
 	"application/vnd.hbci": "Homebanking Computer Interface (HBCI)",
-	"application/vnd.hp-jlyt": "HP Indigo Digital Press - Job Layout Languate",
+	"application/vnd.hp-jlyt": "HP Indigo Digital Press - Job Layout Language",
```

Comment:
There's a difference of opinion about this, but enough people agree that the correct spelling applies.